cli/src/main/java/uk/ac/ebi/atlas/cli/analytics/BioentityPropertiesMapCommand.java

```diff
@@ -42,7 +45,14 @@ public Integer call() {
         } else {
             bioentityPropertiesMapWriter.writeMap(ImmutableSet.copyOf(exclusiveOptions.experimentAccessions), outputFilePath);
         }
+        List<String> failedAccessions = bioentityPropertiesMapWriter.getFailedAccessions();
+        int status = 0;
+        if (failedOutputPath != null && !failedAccessions.isEmpty()) {
+            AccessionsWriter writer = new AccessionsWriter(failedOutputPath, failedAccessions);
+            writer.write();
+            status = 1;
```

Comment:
This status should be 1 regardless of having an output path or not for the failed accessions. The same on the other classes. This is leading to a silent error...

Reply:
You can always be more explicit and do `return 1;`. I don’t have strong opinions on either way.

Reply:
This now separates the action of failing from receiving a file to dump the failed accessions. Previously it was only failing if a requests was made for those accessions and there were failures. Now it will fail only based on whether there are failures or not.